releasetool/commands/tag/nodejs.py

```diff
@@ -29,6 +29,7 @@
 nodejs_docs_only = [
     "googleapis/nodejs-billing-budgets",
     "googleapis/nodejs-datacatalog",
+    "googleapis/nodejs-game-servers",
```

Comment:
@bcoe are we ready to flip everyone over here?

Reply:
we're ready, but there's a somewhat finicky refactor that will be needed in `autorelease` to make it possible, I will need to figure it out with @busunkim96.

Reply:
tldr; I wouldn't block @telpirion on this, but it would be good to figure out how to see this work over the finish line.